File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/InvalideringSed.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.RequiredArgsConstructor;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
@NoArgsConstructor
@RequiredArgsConstructor
public class InvalideringSed {

    private String utstedelsesdato;
    private String type;
    private Grunn grunn;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/InvalideringSed.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class InvalideringSed(
    var utstedelsesdato: String? = null,
    var type: String? = null,
    var grunn: Grunn? = null
)
